### PR TITLE
Fix typo `CRED` to `CRUD`

### PR DESCRIPTION
**Pull request by astraw99**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Fix typo `CRED` to `CRUD`, to better the doc.

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/client-go/tools/cache/delta_fifo.go [hunk 1]
```diff
@@ -146,7 +146,7 @@ const (
 	// as well. Hence, Replaced is only emitted when the option
 	// EmitDeltaTypeReplaced is true.
 	Replaced DeltaType = "Replaced"
-	// Sync is for synthetic events during a periodic resync.
+	// Sync is for synchronous events during a periodic resync.
```
Comment (liggitt) [inline staging/src/k8s.io/client-go/tools/cache/delta_fifo.go:149]: "synthetic" was correct, drop this change

Comment (astraw99) [reply staging/src/k8s.io/client-go/tools/cache/delta_fifo.go]: Got it, fixed. PTAL thanks.

**Pull request merged**